### ISSUE TITLE
Systems that should remove prefixes from shields

### ISSUE THREAD

**Issue by jteresco** [TravelMapping/Web#586]
In the shield generator, we remove prefixes ("NY30" shows up with just "30" in the shield) for many systems.  Others include them.  Please note which systems have prefixes currently displayed in the shield that should have them stripped out.

**Comment by jteresco** [comment]
Looking for guidance on these.  Do we include in the number on the shield...

"RTR" on afrrtr?
"EA" on afrea?
"TAH" on afrtah?
"AH" on asiah and asiahr?
"M" on asim and cism?
"D" on gufd?
"A" and "T" on jama, jamt?

I'll stop at j for now, but you get the idea.

**Comment by si404** [comment]
> "AH" on asiah and asiahr?
> "M" on asim and cism?
> "D" on gufd?
> "A" and "T" on jama, jamt?

yes on all of these (all French roads have the prefixes). The African ones I've not yet seen a sign for - leave the prefixes in.

Australia needs them stripped from ausn, ausnb (white shields)
ausnt, ausqld, ausvicmr, auswa, ausnt (blue shields)
ausqldmr (white hexagons)
ausnswtd, ausqldtd, ausvictd, auswatd (brown pentagons - and some of the Queensland ones have logos rather than the abbreviation of the name)

indjt doesn't have the prefixes, nor does finst.

I mean to go through various systems that use cartouches that don't have them and say what they need.

**Comment by michihdeu** [comment]
It is not just with or without prefix but with or without space (or hyphen) between prefix and number

**Comment by michihdeu** [comment]
I had sorted all non-NA systems by the type of handling (indicated in comment):
https://github.com/TravelMapping/Web/blob/master/shields/shieldgen.php#L184
- use as-is (only replace placeholder)
- use wide svg files
- use wide svg file for 3-digit numbers
- use wide svg files for 3-/4-digit numbers
- use wide svg files for 3-/4-/5-digit numbers
- use wide svg file for 4-digit numbers
- use wide svg file for 5-/7-digit numbers
- remove prefix, use wide svg files for 2-/3-digit numbers
- remove prefix, use wide svg file for 3-digit numbers
- remove prefix, use wide svg file for 4-digit numbers
- remove prefix
- remove prefix and suffix
- add blank after prefix
- add blank after prefix, use wide svg files
- add hyphen after prefix, use wide svg files

**Comment by si404** [comment]
@michihdeu Not all non-NA systems, but an awful lot of them at least. All the ones that use cartouches.

@jteresco I made a post with some cartouche descriptions (in a thread where there's loads). If you want more, then it will have to be the morning my time at the earliest.

**Comment by jteresco** [comment]
Thanks, there's plenty for me to work through here and over in the forum.  I'm not sure I'll be able to make new shapes, so many will probably remain on the plain white squares for now.

**Comment by cockroachking** [comment]
AFAICT it this point all systems that have shields added are correct in terms of prefixes. The same cannot be said for the systems that use the default shield, but I hope to get a unique shield for every (or almost every) system, even if it is something like the usasf "BGS" template.

The systems in this issue mostly seem to have already been addressed (either by me or before), or are Australian, which have their own issue (#595), we can probably close this one.
